presto-tests/src/main/java/com/facebook/presto/tests/StructuralTestUtil.java

```diff
@@ -83,6 +84,19 @@ public static Block arrayBlockOf(Type elementType, Object... values)
         return blockBuilder.build();
     }
 
+    public static Block mapBlockOf(Type keyType, Type valueType, Map<?, ?> value)
```

Comment:
Is this being used anywhere?

Reply:
It is used in a  presto-facebook PR

Reply:
https://github.com/facebookexternal/presto-facebook/pull/1949

Reply:
The Map's iteration order is undefined and it varies from one map implementation to the other.  Using this construct might result in more confusion.

Can the test use ? public static Block mapBlockOf(Type keyType, Type valueType, Object[] keys, Object[] values) which has more predictable ordering.

Reply:
The test cases don't care about order of entries inside the output mapBlock after merging. We convert the output mapBlock in the test case to a Map and compare it to the expected Map

Reply:
Should it just use the Object[] overloads then ?